v4/integrations/nrgrpc/nrgrpc_client_test.go

```diff
@@ -74,14 +74,7 @@ func TestGetURL(t *testing.T) {
 }
 
 func testApp() integrationsupport.ExpectApp {
-	return integrationsupport.NewTestApp(replyFn, integrationsupport.ConfigFullTraces)
-}
-
-var replyFn = func(reply *internal.ConnectReply) {
-	reply.SetSampleEverything()
-	reply.AccountID = "123"
-	reply.TrustedAccountKey = "123"
-	reply.PrimaryAppID = "456"
+	return integrationsupport.NewTestApp(integrationsupport.ConfigFullTraces)
```

Comment:
The signature of `NewTestApp` was changed to remove the `replyFn` argument. This was done in a previous PR.